### ISSUE TITLE
The "path" argument must be of type string

### ISSUE THREAD

**Issue by keeliang** [Pana/nrm#85]
internal/validators.js:124
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

[TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
  at validateString (internal/validators.js:124:11)
  at Object.join (path.js:375:7)
  at Object.<anonymous> (C:\Users\username\AppData\Roaming\npm\node_modules\nrm\cli.js:17:20)
  at Module._compile (internal/modules/cjs/loader.js:1063:30)
  at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
  at Module.load (internal/modules/cjs/loader.js:928:32)
  at Function.Module._load (internal/modules/cjs/loader.js:769:14)
  at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
  at internal/main/run_main_module.js:17:47
] {
  code: 'ERR_INVALID_ARG_TYPE'
}

**Comment by keeliang** [comment]
I modified'process.env.HOME' to'process.env.HOMEPATH', it is OK.

**Comment by ByteAKA8bit** [comment]
Same problem, and it works, but why?

**Comment by jihainan** [comment]
> I modified'process.env.HOME' to'process.env.HOMEPATH' in cli.js, it is OK.

It works!

**Comment by dizys** [comment]
Same problem on Windows 10.

**Comment by hellorayza** [comment]
> > I modified'process.env.HOME' to'process.env.HOMEPATH' in cli.js, it is OK.
> 
> It works!

It doesn't work for me. 
My node was installed by nvm.
Environment:
```
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.18363.1441
```

**Comment by PGambler** [comment]
改了之后是可以的，但是更新一下有回去'process.env.HOME'了，所以我搞了个系统变量'home=%path%'，这不报错了，但是我有切换不了npm源了。。。

it only work once, when restart my computer, it change back to 'process.env.HOME'
so i create a global env : 'home=%path%', this will work;
but i get a new trouble, i can't use 'nrm use XXX' switch my npm...

**Comment by EmilyMew** [comment]
Resolved by PR#83
